flash_px4.sh

```diff
@@ -1 +1,5 @@
-./px_uploader.py --port /dev/ttyACM0 /tmp/ark_fmu-v6x_default.px4
\ No newline at end of file
+#!/bin/bash
+
```

Comment:
We could also add driving the vbus pin and toggling the reset pin here. Just in case the mavlink restart command doesn't work.

Reply:
good idea, I'll test it and add